queue/persistence.go

```diff
@@ -265,10 +277,18 @@ func (q *Persistence) acknowledge(id uint64) {
 }
 
 // Close closes this queue
-func (q *Persistence) Close() error {
+func (q *Persistence) Close(clean bool) error {
 	q.log.Info("queue is closing")
```

Comment:
q.log.Info("queue is closing", log.Any("clean", clean)

Reply:
OK